plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java

```diff
@@ -194,6 +196,26 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Override
+    @Test
+    public void testCountDistinctWithStringTypes()
```

Comment:
Why overridden?

Reply:
Phoenix requires the `ROWKEYS` parameter, I'll add a comment.